src/core/numerics/ohm/ohm.h

```diff
@@ -239,7 +254,7 @@ namespace core
         template<typename VecField, typename ComponentTag>
         auto resistive_(VecField const& J, MeshIndex<VecField::dimension> index, ComponentTag) const
         {
-            auto const eta = 1.0; // TODO : eta should comme from input file
+            auto const eta = this->eta_;
```

Comment:
Possibly unnecessary copy?

Reply:
just because `eta` is shorter than `this->eta_`

Reply:
sorry, I meant the lack of ref(&)

```
auto const& eta = this->eta_;
```

I think copying doubles is super fast anyway so maybe it's not all that important.